TROUBLESHOOTING.md

```diff
@@ -190,3 +191,23 @@ ENOGIT
 ### Related Issues
 
 * [#11095](https://github.com/npm/npm/issues/11095)
+
+## Cannot Read property
+If your install fails and it says something like `Cannot read property 'path' of null`, then it is most likely a network issue in disguise.
```

Comment:
Maybe narrow it to just `Cannot read property 'path' of null`? Have you ever seen an instance of that where the problem is not with the internet?